README.rst

```diff
@@ -0,0 +1,141 @@
+
+
+PDFmorph
+========================================================================
+
+PDFmorph is a Python software package designed to increase the insight
+researchers can obtain from measured atomic pair distribution functions 
+(PDFs) in a model-independent way. The program was designed to help a 
+researcher  answer the question: "has my material undergone a phase 
+transition between these two measurements?"
+
+When given two PDFs to compare, PDFmorph attempts to correct for benign 
+effects like thermal expansion and increased thermal motion before 
+"morphing" one PDF to a selected "target" (usually, the higher-temperature
+PDF). After "stretching", "smearing", and "scaling" the PDF, PDF will
+find the best fit between the morphed and target PDFs and plot them on 
+top of each other with the difference curve plotted below.
+
+Finally, we note that PDFmorph should work on other spectra that are not 
+PDFs, though it has not been extensively tested beyond the PDF.
+
+For more information on PDFmorph, please consult the documentation.
+
+
+LICENSE
+------------------------------------------------------------------------
+
+This software is subject to license and copyright restrictions listed
+`here. <https://github.com/diffpy/diffpy.pdfmorph/blob/master/LICENSE.txt/>`_
+
+
+REQUIREMENTS
+------------------------------------------------------------------------
+
+The PDFmorph application has a Command Line interface. If you are 
+unfamiliar with the terminal or windows command prompt, it is recommended
+that you consult online resources and become somewhat familiar before
+using PDFmorph.
+
+PDFmorph requires Python 3 and several third party libraries 
+that are used by PDFmorph and its components.
+
+* NumPy              - library for scientific computing with Python
+* matplotlib         - Python 2D plotting library
+* SciPy              - library for highly technical Python computing
+* diffpy.utils       - `shared helper utilities <https://github.com/diffpy/diffpy.utils/>`_ for wx GUI
+
+To install these packages, please see installation instructions at 
+their respective web-pages.
+
+
+INSTALLATION
+------------------------------------------------------------------------
+
+To create and activate a conda environment to use this software, run 
+the following command from the command line ::
+	
+	conda create -n pdfmorph_env python=3
+	source activate pdfmorph_env
+
+If you're using Windows, replace ``source activate pdfmorph`` with ::
+	
+	activate pdfmorph_env
+
+When you are finished with the session, exit the environment by running :: 
+
+	source deactivate pdfmorph_env
+
+or ::
+
+	deactivate pdfmorph_env
+
+If you use this conda environment, make sure to run the ``source activate 
+pdfmorph_env`` or ``activate pdfmorph_env`` commands from your command 
+line before every session using the applicaiton.
+
+If you're using Anaconda Python, you can install from the "conda-forge" 
```

Comment:
Anaconda is not a type of python so this text is not quite right.  Conda is a virtual environment system and the text above indicates that people are already using it.  Please see my comments on the website text for a better way to handle this.

Reply:
> see my comments on the website text 

I don't know exactly where you mean, could you clarify?

Reply:
I am not sure if we should put instructions to create new Anaconda environment.
As far as I can tell new environment is not really needed for installing pdfmorph, so such instruction might be more confusing than helpful.  PDFmorph is to my understanding compatible with both Python 2.7 so there is also no need to ask the users to use Python 3.

I would suggest to keep Anaconda-related instructions at minimum as we don't want to repeat (and later maintain) conda manual.  I would also vote to upload pdfmorph releases to the `diffpy` channel and recommend to install it from there as well as from conda-forge.
Can we model the instructions more after what is in
https://github.com/diffpy/diffpy.structure/blob/master/README.rst ?

Installing from `conda-forge` has a side-effect of replacing stock Anaconda Python and a lot of other packages.  Perhaps we should not recommend that as the first choice, especially if user  only wants to add pdfmorph, for which the default Anaconda packages work OK.

Reply:
@pavoljuhas I'm pretty sure @CJ-Wright and I talked about PDFmorph only being compatible with python 3 when we submitted it for release through conda-forge, so to my understanding it isn't compatible with 2.7 or prior (also, I was getting some issues testing the software yesterday that were caused by the same thing)

Should we encourage installation directly from the diffpy channel then instead of including conda-forge? 

Again, my understanding is a little bit limited as of right now so @CJ-Wright and @sbillinge input might be a little more insightful than mine!

Reply:
> ... talked about PDFmorph only being compatible with python 3 ...

There is another PR #12 by @chiahaoliu which passes travis tests for Python 2.7, 3.5, 3.6.
I feel that PR is ripe for merge and when done the code would be all compatible.

> Should we encourage installation directly from the diffpy channel then instead of including conda-forge?

I think we should list both channels.  Installation from the `diffpy` channel is better as an add-on to stock Anaconda environment, because it brings in just a couple of packages.  For users who already have `conda-forge` it is good to know that pdfmorph is there as well.

Reply:
Sounds good! I'll make the change once @CJ-Wright and @chiahaoliu confirm on that one. 

If you wouldn't mind checking the most recent commit to see if the change to the installation channels is better, I'd really appreciate it!